pynbody/snapshot/ramses.py

```diff
@@ -44,7 +44,6 @@
     try:
         import multiprocessing
 
-        import posix_ipc
```

Comment:
This should probably have a noqa flag on it - while posix_ipc is not technically needed in this scope, the ability to import it is being tested here